renovate.json

```diff
@@ -32,7 +32,6 @@
     {
       "description": "Group updates to next cut down on PR noise.",
       "excludePackageNames": ["graphene-django-optimizer", "django-timezone-field", "django-taggit", "flake8"],
-      "groupName": "Releases Batched for Next",
       "matchBaseBranches": ["next"]
     },
```

Comment:
Not just removing this entire block?

Reply:
Good point. For some reason I thought that was still needed to hold back the other packages but they are still covered by the other sections.